diff -urN a/Makefile.am b/Makefile.am
--- a/Makefile.am	2024-04-26 13:47:41
+++ b/Makefile.am	2024-05-13 17:13:22
@@ -7,6 +7,7 @@
 		memcheck \
 		cachegrind \
 		callgrind \
+		cfggrind \
 		helgrind \
 		drd \
 		massif \
diff -urN a/configure.ac b/configure.ac
--- a/configure.ac	2024-04-26 13:47:41
+++ b/configure.ac	2024-05-13 17:13:22
@@ -5664,6 +5664,8 @@
    callgrind/callgrind_annotate
    callgrind/callgrind_control
    callgrind/tests/Makefile
+   cfggrind/Makefile
+   cfggrind/tests/Makefile
    helgrind/Makefile
    helgrind/tests/Makefile
    drd/Makefile
