diff -urN a/Makefile.am b/Makefile.am
--- a/Makefile.am	2024-11-01 02:07:00.000000000 -0300
+++ b/Makefile.am	2024-12-23 16:00:00.222571366 -0300
@@ -7,6 +7,7 @@
 		memcheck \
 		cachegrind \
 		callgrind \
+		cfggrind \
 		helgrind \
 		drd \
 		massif \
diff -urN a/configure.ac b/configure.ac
--- a/configure.ac	2024-11-01 02:07:00.000000000 -0300
+++ b/configure.ac	2024-12-23 16:00:00.222571366 -0300
@@ -5671,6 +5671,8 @@
    callgrind/callgrind_annotate
    callgrind/callgrind_control
    callgrind/tests/Makefile
+   cfggrind/Makefile
+   cfggrind/tests/Makefile
    helgrind/Makefile
    helgrind/tests/Makefile
    drd/Makefile
