diff --git a/Makefile.am b/Makefile.am
index f8c3fc5..dbc43bf 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -7,6 +7,7 @@ TOOLS = \
 		memcheck \
 		cachegrind \
 		callgrind \
+		cfggrind \
 		helgrind \
 		drd \
 		massif \
diff --git a/configure.ac b/configure.ac
index a3584c9..37e22f0 100755
--- a/configure.ac
+++ b/configure.ac
@@ -4949,6 +4949,8 @@ AC_CONFIG_FILES([
    callgrind/callgrind_annotate
    callgrind/callgrind_control
    callgrind/tests/Makefile
+   cfggrind/Makefile
+   cfggrind/tests/Makefile
    helgrind/Makefile
    helgrind/tests/Makefile
    drd/Makefile
