diff -urN a/configure.ac b/configure.ac
--- a/configure.ac	2023-04-28 12:28:21.000000000 -0300
+++ b/configure.ac	2023-06-02 09:34:33.070245048 -0300
@@ -5418,6 +5418,8 @@
    callgrind/callgrind_annotate
    callgrind/callgrind_control
    callgrind/tests/Makefile
+   cfggrind/Makefile
+   cfggrind/tests/Makefile
    helgrind/Makefile
    helgrind/tests/Makefile
    drd/Makefile
diff -urN a/Makefile.am b/Makefile.am
--- a/Makefile.am	2023-01-07 11:48:51.000000000 -0300
+++ b/Makefile.am	2023-06-02 09:34:33.066245092 -0300
@@ -7,6 +7,7 @@
 		memcheck \
 		cachegrind \
 		callgrind \
+		cfggrind \
 		helgrind \
 		drd \
 		massif \
