diff --git a/Makefile.am b/Makefile.am
index 7749442..fa9962a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -7,6 +7,7 @@ TOOLS = \
 		memcheck \
 		cachegrind \
 		callgrind \
+		cfggrind \
 		helgrind \
 		drd \
 		massif \
diff --git a/configure.ac b/configure.ac
index b851798..a7f1fef 100755
--- a/configure.ac
+++ b/configure.ac
@@ -5164,6 +5164,8 @@ AC_CONFIG_FILES([
    callgrind/callgrind_annotate
    callgrind/callgrind_control
    callgrind/tests/Makefile
+   cfggrind/Makefile
+   cfggrind/tests/Makefile
    helgrind/Makefile
    helgrind/tests/Makefile
    drd/Makefile
