diff --git a/Makefile.am b/Makefile.am
index 08db834..7c91f54 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -7,6 +7,7 @@ TOOLS = \
 		memcheck \
 		cachegrind \
 		callgrind \
+		cfggrind \
 		helgrind \
 		drd \
 		massif \
diff --git a/configure.ac b/configure.ac
index 524230d..839ab54 100755
--- a/configure.ac
+++ b/configure.ac
@@ -4816,6 +4816,8 @@ AC_CONFIG_FILES([
    callgrind/callgrind_annotate
    callgrind/callgrind_control
    callgrind/tests/Makefile
+   cfggrind/Makefile
+   cfggrind/tests/Makefile
    helgrind/Makefile
    helgrind/tests/Makefile
    drd/Makefile
